ai-ml/jupyterhub/variables.tf

```diff
@@ -55,6 +55,27 @@ variable "acm_certificate_domain" {
 }
 variable "jupyterhub_domain" {
   type        = string
-  description = "Enter sub-domain name for jupyterhub to be hosted,  e.g. eks.example.com. Only needed if auth mechanism is set to cognito"
+  description = "Enter sub-domain name for jupyterhub to be hosted,  e.g. eks.example.com. Only needed if auth mechanism is set to cognito or oauth"
+  default     = ""
+}
+variable "oauth_domain" {
```

Comment:
nit: Add a new line after each variable declaration for consistency. Same for all the new variables mentioned below